src/assets/scss/components/statusContainer.scss

```diff
@@ -0,0 +1,106 @@
+.status-container {
+  background-color: $off-white;
+  border-radius: 5px;
+  box-shadow: 0px 0px 1px 2px rgba(85, 3, 79, 0.1);

```

Comment:
it seems that you removed the entire background for the columns. can you please revert this change back and just add the changes suggested? it will be easier to visualize as columns.